js/index.js

```diff
@@ -27,3 +27,14 @@ if (hacker1NameLength > hacker2NameLength) {
 // Iteration 3: Loops
 let hacker1InCapital = hacker1.toUpperCase()
 console.log(hacker1InCapital);
```

Comment:
In iteration 3.1 task asked to have a space between letters :) 

One approach is to iterate through the length of the word, convertin every letter to upper case + adding space.

You can review solution uploaded on student portal if still not clear :)